locales/fr/json.json

```diff
@@ -35,14 +35,14 @@
     "Network Authentication Required": "Authentification réseau requise",
     "Network Connect Timeout Error": "Erreur de délai de connexion réseau",
     "Network Read Timeout Error": "Erreur de délai de lecture réseau",
-    "No Content": "Pas content",
+    "No Content": "Pas de contenu",
     "Non-Authoritative Information": "Informations non autorisées",
     "Not Acceptable": "Pas acceptable",
     "Not Extended": "Non prolongé",
     "Not Found": "Page non trouvée",
     "Not Implemented": "Pas mis en œuvre",
     "Not Modified": "Non modifié",
-    "OK": "D'accord",
+    "OK": "OK",
```

Comment:
In this case, you will need to create a `_excludes.json` file and add a value to it:

```json
[
    "OK"
]
```

Reply:
Updated